packages/config/src/balance/balance.interfaces.ts

```diff
@@ -46,18 +50,19 @@ export interface TokensBalanceConfig<
   pallet: BalancePallet.Tokens;
   function: BalanceFunction.Accounts;
   path: [];
-  getParams: (account: string) => [
-    string,
-    (
-      | {
-          Token: Symbols | 'MOVR' | 'KUSD';
-        }
-      | { ForeignAsset: AssetId }
-    ),
-  ];
+  getParams: (account: string) => [string, TokensBalanceParamAsset<Symbols>];
   calc: (data: TokensPalletAccountData) => bigint;
 }
 
+export type TokensBalanceParamAsset<Symbols extends AssetSymbol = AssetSymbol> =
+
+    | {
+        [BalanceCurrencyTypes.Token]: Symbols | 'MOVR' | 'KUSD';
```

Comment:
MOVR and KUSD are included in Symbols

Reply:
Seems like resolving this isn't so simple. We might want to spin this into it's own ticket